src/main/java/com/spotify/logging/LoggingConfigurator.java

```diff
@@ -458,4 +461,15 @@ private static int getSyslogPort() {
     return isNullOrEmpty(port) ? -1 : Integer.valueOf(port);
   }
 
+  private static String getHeliosHostname() {
+    for (Map.Entry<String, String> entry : getenv().entrySet()) {
+      if (entry.getKey().matches("HELIOS_PORT_\\w+")) {
```

Comment:
This environment variable is only present if the helios job has a port mapping.

Perhaps it would make more sense for Helios to send something like `SPOTIFY_HOSTNAME` for all types of jobs which would allow this patch to work 100% of the time.

Also by making the name more generic, it could be used by systems other than Helios as well (if that need ever arises).

I'll look into getting this done with helios.